client/src/components/mapBox.js

```diff
@@ -1,35 +1,45 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+
 import React from 'react';
 import PropTypes from 'prop-types';
 import InfoItem from './infoItem';
 
 import map from '../images/map.png';
-import iconAddress from '../images/icon-address.png';
-import iconDirections from '../images/icon-get-directions.png';
-import iconPhone from '../images/icon-business-phone.png';
-import iconBusinessUrl from '../images/icon-business-url.png';
-import iconReservation from '../images/icon-reservation.png';
-import iconSendPhone from '../images/icon-send-to-phone.png';
 
+const png = {
```

Comment:
Interesting way of storing svg paths

Reply:
It's a little ugly, and the longest line is 500+ characters. But, I think it works. I considered moving the paths to a separate, helper data file to keep the component file cleaner. But, it's the only instance of this kind. So, I skipped it. Should probably refactor to a separate file. It's on my list.